ACPs/TEMPLATE.md

```diff
@@ -1,13 +1,11 @@
-```text
-ACP: <PR Number>
-Title: <ACP title>
-Author(s): <a list of the author's name(s) and optionally contact info: FirstName LastName <foo@bar.com>>
-Discussions-To: <GitHub Discussion URL (POPULATED BY MAINTAINER, DO NOT SET)>
-Status: <Proposed, Implementable, Activated, Stale>
-Track: <Standards, Best Practices, Meta, Subnet>
-Replaces (*optional): <ACP number>
-Superseded-By (*optional): <ACP number>
-```
+| ACP                            | PR Number                                                                                                                                           |
+| :--- | :--- |
+| **Title**                      | ACP title                                                                                                                                           |
+| **Author(s)**                  | A list of the author's name(s) and optionally contact info: FirstName LastName ([@GitHubUsername](./README.md) or [email@address.com](./README.md)) |
+| **Status**                     | Proposed, Implementable, Activated, Stale ([Discussion](./README.md))                                                                               |
+| **Track**                      | Standards, Best Practices, Meta, Subnet                                                                                                             |
+| **Replaces (\*optional)**      | [ACP-XX](./README.md)                                                                                                                               |
+| **Superseded-By (\*optional)** | [ACP-XX](./README.md)                                                                                                                               |
```

Comment:
```suggestion
| ACP | PR Number |
| :- | :- |
| **Title** | ACP title |
| **Author(s)** | A list of the author's name(s) and optionally contact info: FirstName LastName (link to GitHub profile or email address) |
| **Status** | Proposed, Implementable, Activated, Stale |
| **Discussion** | [POPULATED BY MAINTAINER. DO NOT SET] |
| **Track** | Standards, Best Practices, Meta, Subnet |
| **Replaces (_optional_)** | [ACP-XXX](./XXX-name/README.md) |
| **Superseded-By (_optional_)** | [ACP-XXX](./XXX-name/README.md) |
```

Nits: I think we should keep the discussion link separate from the Status, can remove extra whitespace, and make it clear the links are templates that aren't actually valid.

Reply:
The discussion link matter was discussed here https://github.com/avalanche-foundation/ACPs/discussions/86#discussioncomment-9500173

I can remove the whitespaces yes. For the links, I think that people will figure it out by themselves.

Reply:
Ah thanks, was missing that context